charts/fybrik/templates/manager-deployment.yaml

```
@@ -34,6 +34,20 @@ spec:
       serviceAccountName: {{ .Values.manager.serviceAccount.name }}
       securityContext:
         {{- toYaml .Values.manager.podSecurityContext | nindent 8 }}
+      {{- if .Values.manager.solver.image }}
```

```suggestion
      {{- if .Values.manager.solver.enabled}}
```
It's the `enabled` flag we want to depend on, isn't it?
Note that this `if` statement appears 3 more times

---

We want to deploy once and be able to switch usage on the fly by changing the configmap manually.